pkg/locker/locker_api.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package locker
+
+// package locker provides security API for protecting data for storage and/or transfer
+
+// Locker is a service that protects a message
+type Locker interface {
+	// Lock a message msg into ciphertext
```

Comment:
could you plz specify that we are locking key and not the message?

Reply:
I was thinking about this, having this locker is actually redundant. It adds 2 things not found in aries-framework-go:
use a simple `[]byte` for the message argument as opposed to a request struct and the implementation exposes a Shamir constructor.

I think the service is not really needed, I'll create a service that returns Shamir splitting instead. The locker should be one coming from aries-framework-go to allow passing it to a KMS constructor. This new Locker interface is not compatible with secretlock.Service in the framework.